test/integration/builder/inserts.js

```diff
@@ -647,6 +647,37 @@ module.exports = function(knex) {
       });
     });
 
+
+    it('#757 - knex.batchInsert(tableName, bulk, chunkSize)', function (done) {
+      this.timeout(20000);
```

Comment:
Just to make sure that batch insert does what it is supposed to dataset could be e.g. 100 items, which are added in batch of 30 items or so? We need to avoid long running tests.